src/SmartTransactionsController.ts

```diff
@@ -718,38 +775,35 @@ export default class SmartTransactionsController extends StaticIntervalPollingCo
       );
     }
     transactions.push(unsignedTradeTransactionWithNonce);
-    const data = await this.fetch(getAPIRequestURL(APIType.GET_FEES, chainId), {
-      method: 'POST',
-      body: JSON.stringify({
-        txs: transactions,
-      }),
-    });
-    let approvalTxFees;
-    let tradeTxFees;
+    const data = await this.#fetch(
+      getAPIRequestURL(APIType.GET_FEES, chainId),
+      {
+        method: 'POST',
+        body: JSON.stringify({
+          txs: transactions,
+        }),
+      },
+    );
+    let approvalTxFees: IndividualTxFees | null = null;
+    let tradeTxFees: IndividualTxFees | null;
```

Comment:
Should both variables be initialized to `null`?

```suggestion
    let approvalTxFees: IndividualTxFees | null = null;
    let tradeTxFees: IndividualTxFees | null = null;
```

Reply:
actually this is an old code I moved the assignment to the statement below